### ISSUE TITLE
Issue with forecast.plot(): urlopen error [Errno -3]

### ISSUE THREAD

**Issue by khawajasim** [SCECcode/pycsep#141]
Hi,

I was plotting forecasts, which was working fine until recently. But for some time it is giving this error.

URLError: <urlopen error [Errno -3] Temporary failure in name resolution>.

This is I guess due to not working of this link:
https://naciscdn.org/naturalearth/110m/physical/ne_110m_coastline.zip

Can you plot some forecast and confirm whether there is error or I am not doing it right.

Kind Regards

**Comment by wsavran** [comment]
sure i can look into this more. i tried to go to that link and it looks like its offline, so i think that is what is causing your error. we should add error handling for this case. thanks for pointing this out.

**Comment by wsavran** [comment]
this issue appears to be fixed with newer versions of cartopy. try running 
    conda update cartopy 
and rerunning the script. in general, its good practice to update your conda packages regularly.

**Comment by wsavran** [comment]
adding some additional information here: from this conversation: https://github.com/SciTools/cartopy/issues/1879

tl;dr. the new version of cartopy 0.2.0 requires proj8 which does not have a package available on apt for debian linux systems. this can be problematic for our CI tools. i'm going to leave this open for now and update when the > proj8.0.0 gets released on the stable branch of aptitude package manager.

**Comment by wsavran** [comment]
@khawajasim did updating cartopy in conda work for you?

**Comment by wsavran** [comment]
thats great! do you mind closing this issue when you are happy that it works?
